underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -347,13 +347,13 @@ public void setMode(String path, short mode) throws IOException {}
   // Not supported
```

Comment:
I see, could you elaborate in the comment here to make this clear? i.e., we always return empty strings here are as owner/group, and propagating no more info from gcs/s3/etc